packages/fxa-settings/src/pages/Signin/SigninRecoveryChoice/index.tsx

```diff
@@ -137,12 +137,12 @@ const SigninRecoveryChoice = ({
   const cmsInfo = integration?.getCmsInfo();
   const cmsButton = {
     color: cmsInfo?.shared?.buttonColor,
-  }
+  };
 
   return (
     <AppLayout cmsInfo={cmsInfo}>
       <div className="relative flex items-center mb-5">
-        <ButtonBack />
+        <ButtonBack cmsBackgroundColor={cmsInfo?.shared?.backgroundColor} />
```

Comment:
Should this also be applied to the `BackButton` in `SigninRecoveryCode` and `SigninRecoveryPhone`?

Reply:
Not sure of the best way to make sure we also extend this to other pages as we expand the use of cmsInfo - for example, to the FlowContainer in the inline 2FA setup flows 🤔

Reply:
Ah lemme look into this, you're probably right. I will need a follow up issue to add this in content-server though.

Yeah if we're finding ourselves prop drilling CMS related things down, we could consider adding it to Context.